package.json

```diff
@@ -1,11 +1,11 @@
 {
   "private": true,
   "scripts": {
-    "parse-package-set": "spago -x integration.dhall run",
-    "bench-file": "spago -x bench.dhall build && node --expose-gc -e \"require('./output/BenchFile/index.js').main()\""
+    "integration": "spago -x integration/integration.dhall run",
```

Comment:
FWIW, I'm not sure why this is called "integration" instead of everything being called "parse-package-set" since the latter is much more explicit about what's happening 😄